src/hotspot/cpu/aarch64/assembler_aarch64.hpp

```diff
@@ -3138,6 +3140,20 @@ void mvnw(Register Rd, Register Rm,
   INSN(sve_st1d, 0b1110010, 0b11, 0b111, 0b010);
 #undef INSN
 
+// SVE load gather, store scatter (scalar plus vector) - 32-bit scaled offset
```

Comment:
In the Arm ARM Supplement for SVE, this group is called "Gather/scatter load/store/prefetch (SVE)".